editor/editor_settings.cpp

```diff
@@ -821,6 +821,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("run/output/always_open_output_on_play", true);
 	_initial_set("run/output/always_close_output_on_stop", false);
 
+	_initial_set("run/output/max_lines", 10000);
```

Comment:
Add a valid range:

```suggestion
	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_RANGE, "run/output/max_lines", 10000, "100,100000,1")
```

Very low values (below 100) aren't really useful, and very high values (above 100,000) can cause issues such as the editor becoming pretty much unusable, so I suggest clamping within this range.

Reply:
Change made, will be in next commit amend.